task9.py

```diff
@@ -0,0 +1,96 @@
+from datetime import datetime
+import sqlite3
+
+
+class LoggerMetaclass(type):
+    INFO = 10
+    WARNING = 20
+    ERROR = 30
+    NOTSET = 0
+    level = NOTSET
+    LOGFILE = 'logs.txt'
+    f = open(LOGFILE, 'a', encoding='utf-8')
```

Comment:
Rename **f** to log_file (f so similar to f-string)
Move in log_file and connection in __new__